code_curation2_submissions/Santiage_rs_seed_repo_generator.m

```diff
@@ -1,54 +1,36 @@
 function [Result] = rs_seed_repo_generator(Seeds,BOG_rsMRI,PSN_rsMRI,MainDataFolder,PipelineName)
 % function [ADataDir,AMaskFilename, AROIDef,AResultFilename, ACovariablesDef, VorR_tag, MainOutputFolder] = seed_repo_generator(Seeds,BOG_rsMRI,PSN_rsMRI,MainDataFolder,PipelineName)
-
-
-%%Function that creates the necessary input arguments for the function
-%%rp_fc which generates FC maps. The maps are then saved in a repository seed
-%%folder according to specific ROIs per BOGEN_ID.
-
-% This function is made to work in conjunction  (within the same loops) with
-% the rp_fc function, which is part of the RESTplus_1.2 toolbox.
-
-
-% INPUTS
-
-% - Seeds             = jx4 numeric array where each seed j is represented
-%                       by x,y,z MNI coordinates and radius in mm, respectively
-
-% - BOG_rsMRI         = string that contains the bogen id of the rs MRI
-% - PSN_rsMRI         = string that contains the PSN of the rs MRI
-% - MainDataFolder    = string that contains the path to the data
-% - PipelineName      = string that contains the full name of the folder in which the pipeline is ran
+% Function that creates the necessary input arguments for the function rp_fc which generates FC maps. 
+% The maps are then saved in a repository seed folder according to specific ROIs per BOGEN_ID.
+% This function is made to work in conjunction  (within the same loops) with the rp_fc function, which is part of the RESTplus_1.2 toolbox.
+%
+% INPUTS:
+% - Seeds             = jx4 numeric array, each seed j is represented by x,y,z MNI coordinates and radius in mm, respectively
+% - BOG_rsMRI         = string, the bogen id of the rs MRI
+% - PSN_rsMRI         = string, the PSN of the rs MRI
+% - MainDataFolder    = string, the path to the data
+% - PipelineName      = string, the full name of the folder in which the pipeline is ran
```

Comment:
Should add an 'Outputs:' section to indicate what this function returns.
If the function does not return anything, then just type:
Outputs:
 None